dev-lang/mono/mono-4.4.1.0.ebuild

```diff
@@ -3,10 +3,8 @@
 # $Id$
 
 EAPI=6
-AUTOTOOLS_PRUNE_LIBTOOL_FILES="all"
-AUTOTOOLS_AUTORECONF=1
```

Comment:
Could you split that into a separate commit? It's not really related to multilib but a generic cleanup.